### PR TITLE
Fix Angular preview

### LINKED ISSUE #34: Angular Preview sometimes broken
**Description**
After a ui schema change, the first time the preview is rendered it's pretty broken:
* doesn't validate properly
* doesn't react on selection changes
* overlays texts etc.

**To reproduce**

1. Go to 'Preview'
2. Enter something into an input

**Workaround**
Switch back to 'Editor' and right back to 'Preview'. Now it works as expected.

**Screenshot**
AngularPreview

### PR DESCRIPTION
**Pull request by AlexandraBuzila**
Updated preview element with version that accepts input as properties. Fixes #34.

### CODE REVIEW

File: src/editor/components/EditorPreview.tsx [hunk 1]
```diff
@@ -21,22 +20,19 @@ declare global {
...
-        schema,
-        uiSchema
-      );
-    }
-  }, [schema, editorUISchema]);
-  return (
+  const inputSchema = JSON.stringify(schema);
+  const inputUISchema = JSON.stringify(uiSchema);
+
+  return inputUISchema ? (
```
Comment (eneufeld) [inline src/editor/components/EditorPreview.tsx:30]: shouldn't we check for `inputSchema`?

Comment (AlexandraBuzila) [reply src/editor/components/EditorPreview.tsx]: I added the check.

File: src/editor/components/EditorPreview.tsx [hunk 2]
```diff
@@ -21,22 +20,19 @@ declare global {
 }
 
 export const EditorPreview: React.FC = () => {
-  const ngJsonForms = useRef<JSX.IntrinsicElements['ng-jsonforms']>(null);
   const schema = useExportSchema();
   const editorUISchema = useUiSchema();
+  const uiSchema = editorUISchema ? buildUiSchema(editorUISchema) : undefined;
```
Comment (sdirix) [inline src/editor/components/EditorPreview.tsx:25]: Can't we use the `useExportUiSchema`?

Comment (AlexandraBuzila) [reply src/editor/components/EditorPreview.tsx]: Completely missed that we have that

**Pull request merged**